rds/rds.go

```diff
@@ -0,0 +1,65 @@
+package rds
+
+import (
+	"context"
+	"fmt"
+	"github.com/ONSdigital/dp-areas-api/config"
+	"github.com/ONSdigital/dp-areas-api/models"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/service/rds/rdsutils"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+type RDS struct {
+	conn *pgxpool.Pool
+}
+
+func (r *RDS) Init(ctx context.Context, cfg *config.Config) error {
+	authToken, err := rdsutils.BuildAuthToken(
+		cfg.GetRDSEndpoint(),
+		cfg.AWSRegion, cfg.RDSDBUser,
+		credentials.NewEnvCredentials(),
+	)
+	if err != nil {
+		return err
+	}
+
+	connStr := fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s",
+		cfg.RDSDBHost, cfg.RDSDBPort, cfg.RDSDBUser, authToken, cfg.RDSDBName,
+	)
+
+	rdsConn, err := pgxpool.Connect(ctx, connStr)
+	if err != nil {
+		return err
+	}
+
+	r.conn = rdsConn
+	return nil
+}
+
+func (r *RDS) Close() {
+	r.conn.Close()
+}
+
+func (r *RDS) ValidateArea(areaCode string) error {
+	area := "select code from area where code = $1"
+	var code string
+	err := r.conn.QueryRow(context.Background(), area, areaCode).Scan(&code)
+	return err
+}
+
+func (r *RDS) GetRelationships(areaCode string) ([]*models.AreaBasicData, error) {
+	var relationships []*models.AreaBasicData
+	relationshipQuery := "select code, name from area, area_relationship where rel_area_code = code and area_code = $1"
```

Comment:
We should have a package for SQL queries, and call them in the method that they're utilised in; a possible location could be:

`github.com/mikek/dp-areas-api/sql/queries.go`

`var GetRelationshipsQuery = "select code, name from area, area_relationship where rel_area_code = code and area_code = $1"`

`relationshipQuery := sql.GetRelationshipsQuery`